### PR TITLE
PCHR-2609: Update main menu css

### PR DESCRIPTION
**Pull request by AkA84**
## Overview
This PR restyles the top main menu, including the submenus and the quicksearch field

### Menu closed
before-menu-closed

### Menu opened
before-menu-opened

### Quicksearch
before-quicksearch

### Quicksearch results
before-quicksearch-results

### Menu closed
menu-closed

### Menu opened
menu-opened

### Quicksearch
quicksearch

### Quicksearch results
quicksearch-results

### CODE REVIEW

File: scss/civicrm/main-menu/_main-menu.scss [hunk 1]
```diff
@@ -1,86 +1,175 @@
...
+  background: $gray-lighter;
+  border: 0;
+  box-shadow: $box-shadow;
+  font-family: $font-family-base;
+  left: 0 !important;
+  padding: $crm-main-menu-sub-menu-padding;
+  top: $crm-main-menu-height !important;
+
+  .ui-menu-item {
+    font-size: 13px;
```
Comment (reneolivo) [inline scss/civicrm/main-menu/_main-menu.scss]: font-size: [$font-size-base](https://github.com/compucorp/org.civicrm.shoreditch/blob/staging/scss/bootstrap/overrides/_variables.scss#L88)

File: scss/civicrm/main-menu/_main-menu.scss [hunk 2]
```diff
@@ -1,86 +1,175 @@
...
     display: none;
   }
 
-  .fa.menu-item-arrow {
+  .menu-div {
+    margin-top: 0 !important;
+  }
+
+  .menu-item {
+    font-size: 13px;
```
Comment (reneolivo) [inline scss/civicrm/main-menu/_main-menu.scss]: font-size: [$font-size-base](https://github.com/compucorp/org.civicrm.shoreditch/blob/staging/scss/bootstrap/overrides/_variables.scss#L88)

File: scss/civicrm/main-menu/_main-menu.scss [hunk 3]
```diff
@@ -1,86 +1,175 @@
...
-#civicrm-menu > .menumain > a,
-#root-menu-div .menu-item,
-#root-menu-div .menu-item a {
-  color: $gray-darker;
+    .form-text {
+      background: transparent !important;
+      border: none;
+      color: $gray-dark;
+      font-family: $font-family-base;
+      font-size: 13px;
```
Comment (reneolivo) [inline scss/civicrm/main-menu/_main-menu.scss]: font-size: [$font-size-base](https://github.com/compucorp/org.civicrm.shoreditch/blob/staging/scss/bootstrap/overrides/_variables.scss#L88)

File: scss/civicrm/main-menu/_main-menu.scss [hunk 4]
```diff
@@ -1,86 +1,175 @@
...
+
+      &:focus {
+        outline: 0;
+      }
+    }
+  }
+
+  > li.menumain {
+    border: 0;
+    font-size: 13px;
```
Comment (reneolivo) [inline scss/civicrm/main-menu/_main-menu.scss]: font-size: [$font-size-base](https://github.com/compucorp/org.civicrm.shoreditch/blob/staging/scss/bootstrap/overrides/_variables.scss#L88)

File: scss/civicrm/main-menu/_main-menu.scss [hunk 5]
```diff
@@ -1,86 +1,175 @@
...
     font-family: $font-family-base;
+    padding: $crm-main-menu-sub-menu-padding;
 
     li {
+      margin: 0 !important;
+      padding: 0 !important;
+    }
+    
+    .menu-separator {
+      border-color: #D3DEE2;
```
Comment (reneolivo) [inline scss/civicrm/main-menu/_main-menu.scss]: border-color: [$crm-dark-grayblue](https://github.com/compucorp/org.civicrm.shoreditch/blob/staging/scss/bootstrap/overrides/_variables.scss#L67)
or
border-color: [$crm-dialog-container-border](https://github.com/compucorp/org.civicrm.shoreditch/blob/staging/scss/bootstrap/overrides/_variables.scss#L71)

**Review by reneolivo — CHANGES_REQUESTED** [review]

**Review by reneolivo — APPROVED** [review]

**Comment by totten** [comment]
The expanded padding on the menu is a pretty visible part of v0.1-alpha14.

Anecdotal feedback (from a long-term desktop user's perspective): the main menu bar feels pretty big, and the subnavs feel better. I'm not quite sure why the main nav feels big... there's no single objective benchmark for "big" or "small".  (Maybe the typical `btn` or `navbar` sizing is a reference?) 

Of course, you might get more people saying, "It's too small!" So just take that as one subjective data-point.

**Comment by totten** [comment]
After using for a day, I really do like the subnavs!

**Comment by AkA84** [comment]
@totten is the taller main menu an acquired taste as well?

**Comment by totten** [comment]
I haven't really acquired that taste yet. ;) Maybe if I switch from a desktop to a mobile device.

**Pull request merged**